app.rb

```diff
@@ -1,35 +1,64 @@
-require 'net/http'
-require 'uri'
-require 'openssl'
-require 'json'
-require 'redis'
-
-redis = Redis.new(:host => ENV["REDIS_HOST"])
-
-# TODO: Query database for each unique cluster fqdn
-# for each cluster fqdn, append port number, endpoint, and query
-fqdn = ['ganeti-psf.osuosl.bak', 'ganeti-civicrm.osuosl.bak']
-fqdn.each do |name|
+require 'sinatra/base'
+require File.expand_path '../environment.rb', __FILE__
+
+# IAM - a resource usage metric collection and reporting system
+class Iam < Sinatra::Base
+  # require the models, but make sure to do it after the test db is migrated
+  require 'models'
+
+  # load up all the plugins
+  Dir[File.dirname(__FILE__) + '/plugins/*.rb'].each { |file| require file }
+
+  ##
+  # static Pages
+  ##
+
+  get '/' do
+    'Hello'
+  end
+
+  ##
+  # Clients
+  ##
+
+  get '/clients' do
+    @clients = Client.each { |x| p x.name }
+    erb :'clients/index'
+  end
+
+  get '/clients/new' do
+    erb :'clients/new'
+  end
+
+  ## TODO - move this somewhere
+  redis = Redis.new(host: ENV['REDIS_HOST'])
+
+  # TODO: Query database for each unique cluster fqdn
+  # for each cluster fqdn, append port number, endpoint, and query
+  fqdn = ['ganeti-psf.osuosl.bak', 'ganeti-civicrm.osuosl.bak']
+  fqdn.each do |name|
```

Comment:
I agree that we should get rid of anything in `app.rb` that isn't necessary. It should just have enough to run the application for testing purposes.